### ISSUE TITLE
Allow to increase size of map in Cat view

### ISSUE THREAD

**Issue by coderbyheart** [NordicSemiconductor/asset-tracker-cloud-app-js#317]
to allow more convenient inspection of larger tracks.

**Comment by FaitAccompli** [comment]
Hi @coderbyheart , I'd like to work on this. By "allow to increase size of map" do you mean adding a full screen control on the cat view?

<img width="77" alt="image" src="https://user-images.githubusercontent.com/15321518/151700201-8cb04e8f-ab9f-4d6d-b2be-7b80bcf40382.png">

**Comment by coderbyheart** [comment]
Yes, that would be one option. I would also like the user to be able to drag the bottom of the map to the desired size while not in full screen.

![image](https://user-images.githubusercontent.com/188915/151769058-0566ca6a-3d3f-47c6-9487-05e736c488a1.png)

**Comment by FaitAccompli** [comment]
I was able to implement something simple. However, the resize css property is by default located at the bottom right portion of the container. We can try implementing hooks to make it more customized and allow the  whole bottom portion to be resizable vertically. But if the one shown below looks good, I'll be more than happy to make a PR.

<img width="312" alt="image" src="https://user-images.githubusercontent.com/15321518/151809913-d2cae829-82a9-4fc6-adf4-00db127b3820.png">

**Comment by coderbyheart** [comment]
I draft PR would have allowed me to try it out, but from the pictures it looks good.
Except, as you mentioned, right now the positioning is not optimal.
Does it work on mobile as well?

**Comment by FaitAccompli** [comment]
I'm currently working on an approach that uses hooks. I'll be testing on mobile as well. Will create a draft PR soon.

**Comment by coderbyheart** [comment]
Note that the map is now in the AWS application https://github.com/NordicSemiconductor/asset-tracker-cloud-app-aws-js/releases/tag/v1.4.0, if you haven't done too much implementation, you could add it there directly.

Otherwise continue in this repo, and I'll port it over.

**Comment by FaitAccompli** [comment]
@coderbyheart, I'm getting an unknown web-app-config error. When following the instructions below. I'm assuming the nRF Asset Tracker for AWS is the aws directory we created based on the Getting Started page. 

![image](https://user-images.githubusercontent.com/15321518/152356412-1826caa5-a617-4fb8-bcc2-882592f13120.png)


[Edit 2/3/2022]
I'm guessing web-app-config is just a placeholder? So in this case we use 

`node cli react-config > ../asset-tracker-cloud-app-aws-js/.envrc`

**Comment by coderbyheart** [comment]
You need to use this branch for the new web app: https://github.com/NordicSemiconductor/asset-tracker-cloud-aws-js/pull/726

**Comment by FaitAccompli** [comment]
Hi @coderbyheart , I created a draft PR #[10](https://github.com/NordicSemiconductor/asset-tracker-cloud-app-aws-js/pull/10)
